graphman-client/modules/help.js

```diff
@@ -23,6 +23,8 @@ module.exports = {
         console.log("        # address - a valid graphman url. Default is https://localhost:8443/graphman");
         console.log("        # username - gateway user for administration.");
         console.log("        # password - gateway user's password for administration.");
+        console.log("        # certFilename - gateway user certificate certificate used for authentication.");
+        console.log("        # certificateCertKey - gateway user certificate key used for authentication.");
```

Comment:
# certFilename - user's certificate for authentication
# keyFilename - user's private key for authentication